admin/model_search_index_create_request.go

```diff
@@ -0,0 +1,199 @@
+// Code based on the AtlasAPI V2 OpenAPI file
+
+package admin
+
+import (
+	"encoding/json"
+)
+
+// SearchIndexCreateRequest struct for SearchIndexCreateRequest
+type SearchIndexCreateRequest struct {
+	// Label that identifies the collection to create an Atlas Search index in.
+	CollectionName string `json:"collectionName"`
+	// Label that identifies the database that contains the collection to create an Atlas Search index in.
+	Database string `json:"database"`
+	// Label that identifies this index. Within each namespace, names of all indexes in the namespace must be unique.
+	Name string `json:"name"`
+	// Type of the index. The default type is search.
+	Type       *string                      `json:"type,omitempty"`
+	Definition *VectorSearchIndexDefinition `json:"definition,omitempty"`
```

Comment:
`Definition` is fixed to VectorSearchIndexDefinition, which is missing properties associated to the TextSearchIndexDefinition view.

Reply:
Fixed. Pushed extra commit to the PR extending this transformation case.